src/pages/AuthPage/Auth.scss

```diff
@@ -0,0 +1,138 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
+// Backgrounds
+$page-background:  url(../../images/bg.jpg) center/cover no-repeat;
```

Comment:
First of all why do u store your images under src folder?
In react u have public folder with assets
Another thing - create background using css is a bad sollution. Better to create div with img and add position absolute
Another thing - jpg is a very bad format for web apps
Another thing - using big images for background it's a bad idea, because the image takes too long toload with a slow internet